hoomd/logging.py

```diff
@@ -210,9 +220,9 @@ def __init__(self, name, cls, category='scalar', default=True):
     def yield_names(self, user_name=None):
         """Infinitely yield potential namespaces.
 
-        Used to ensure that all namespaces are unique for a
-        `hoomd.logging.Logger` object. We simple increment a number at the end
-        until the caller stops asking for another namespace.
+        Used to ensure that all namespaces are unique for a `Logger` object. We
+        simple increment a number at the end until the caller stops asking for
```

Comment:
```suggestion
        simply increment a number at the end until the caller stops asking for
```

Reply:
"simple/simply" is a charged word. In any case "We" aren't doing this, `yield_names` is. I reworded appropriately.